config.yaml

```diff
@@ -4,3 +4,14 @@ options:
     default: '/opt/cni/bin'
     description: |
       Path to install SR-IOV CNI plugin to.
+  image-registry:
+    default: rocks.canonical.com:443/cdk
+    type: string
+    description: |
+      Source registry of SRIOV CNI images.
+      By setting to a value, each image listed in the releases manifest
+      has its image-registry replaced.
+  release:
+    default: v2.6.3
```

Comment:
what happens if you default this to empty or null?  Will it just use the charms' latest version. 

The nice thing about this makes charm upgrades always use the charm's latest version.

Reply:
That makes sense! I'll change it straight away :)